playwright.config.ts

```diff
@@ -37,20 +37,19 @@ export default defineConfig({
     {
       name: "chromium",
       use: {
-        ...devices["Desktop Chrome"],
-        // , headless: false
+        ...devices["Desktop Chrome"], headless: false
```

Comment:
По дефолту `process.env.CI` установлен в `true` на ci, поэтому должно работать без дополнительного прокидывания параметра CI для github
А вот локально не уверен

И тогда джоба должна успешно пройти
```suggestion
        ...devices["Desktop Chrome"], 
        headless: process.env.CI ? true : false
```